src/utils/scroll-to/scroll-to.ts

```diff
@@ -0,0 +1,141 @@
+import { PluginObject } from 'vue';
```

Comment:
Je ne sais pas s'il y aurait des tests unitaires à ajouter sur certaines méthodes? il y a beaucoup de chsoes qui s'approchent pus du UX, donc c'Est peut-être pas pertinent parce qu'il faut un humain pour uger si c'Est correct, mais il y a surements quelques points qui sont pertinents de tester unitairement?

Reply:
Salut , Le problème c'est que la classe utilise beaucoup de service du browser (windows, HTMLElement. etc) tout ces trucs sont accessibles que dans un context browser, j'imagine qu'on pourrait faire des tests automatisé mais ca serait plus efficace d'être end-to-end. Qu'en penses-tu?

Reply:
je vais devoir attendre la rencontre avec le spécialiste des tests qui vient voir l'équipe Brio vendredi pour te répondre, s'il nous donne des conseils à ce sujet..! pour l'instant, c'est difficile, il y a surement quelques fonctions publiques qui auraient des parties testables, mais effectivement le mockage de tout ca sera possiblement trop compliqué.. je pense que c'est plus du UX et possiblement pas pertinent de tester sauf manuellement pour voir l'utilisabilité..